monitoring/package-metadata.json

```diff
@@ -4,7 +4,7 @@
   "description": "A package for monitoring the platform services",
   "type": "infrastructure",
   "version": "0.0.1",
-  "dependencies": ["message-bus-kafka"],
+  "dependencies": [],
```

Comment:
isn't this package is dependent on Kafka minion and can't stand by without it ? 
I think it is better to keep that dependency, what do you think ?

Reply:
`kafka-minion` is in the kafka package and if it doesn't exist it won't be scraped for metrics. I think this dependency is around for the `prometheus-kafka-adapter`. @marrouchi can't that service be removed now that we have the `prometheus-backup`? Then maybe we can get rid of the Kafka dependency.